docs/breaking-changes.md

```diff
@@ -12,6 +12,14 @@ This document uses the following convention to categorize breaking changes:
 * **Deprecated:** An API was marked as deprecated. The API will continue to function, but will emit a deprecation warning, and will be removed in a future release.
 * **Removed:** An API or feature was removed, and is no longer supported by Electron.
 
+## Planned Breaking API Changes (14.0)
+
+### Removed: `worldSafeExecuteJavaScript`
+
+In Electron 14, `worldSafeExecuteJavaScript` will be removed.  There is no alternative, please
+ensure your code works with this property enabled.  It has been enabled by default since Electron
+12.
```

Comment:
Can we add a link to more info about what `worldSafeExecuteJavaScript` does and how to change your code to compensate?

Reply:
The docs for `worldSafeExecuteJavaScript` explain "values returned from `webFrame.executeJavaScript` will be sanitized to ensure JS values can't unsafely cross between worlds when using `contextIsolation`".  I'm not sure how much detail we want here.

Reply:
I think this document should contain enough information to:

1. Enable apps to check whether or not their app will be affected by the change, and
2. Give a brief outline of how to adapt to the change.

For (1) we should mention greppable functions (e.g. `webFrame.executeJavaScript`, maybe also `executeJavaScriptInIsolatedWorld`?). For (2) we don't need to go into a lot of detail, maybe just "ensure that objects returned from `executeJavaScript` are serializable"?